src/LongRunning/OperationsClient.php

```diff
@@ -49,11 +49,11 @@
 use Google\GAX\GrpcCredentialsHelper;
 use Google\GAX\PageStreamingDescriptor;
 use Google\GAX\ValidationException;
-use google\longrunning\CancelOperationRequest;
-use google\longrunning\DeleteOperationRequest;
-use google\longrunning\GetOperationRequest;
-use google\longrunning\ListOperationsRequest;
-use google\longrunning\OperationsGrpcClient;
+use Google\Longrunning\CancelOperationRequest;
+use Google\Longrunning\DeleteOperationRequest;
+use Google\Longrunning\GetOperationRequest;
+use Google\Longrunning\ListOperationsRequest;
+use Google\Longrunning\OperationsGrpcClient;
```

Comment:
I can't remember, have we fixed-up upper-camel casing for other services? (the ideal here would be `LongRunning`)